HISTORY.md

```diff
@@ -6,15 +6,34 @@ dev
 
 - \[Short description of non-trivial change.\]
 
+
+2.28.0 (2022-06-08)
+-------------------
+
+**Improvements**
+
+- Wrap JSON parsing issues in Request's JSONDecodeError for payloads without
+  an encoding to make `json()` API consistent. (#6097)
+- Parse header components consistently, raising an InvalidHeader error in
+  all invalid cases. (#6154)
+- Added provisional 3.11 support with current beta build. (#6155)
+- Requests got a makeover and we decided to paint it black. (#6095)
+
 **Bugfixes**
 
+- Fixed bug where setting `CURL_CA_BUNDLE` to an empty string would disable
+  cert verification. All Requests 2.x versions before 2.28.0 are affected. (#6074)
 - Fixed urllib3 exception leak, wrapping `urllib3.exceptions.SSLError` with
-  `requests.exceptions.SSLError` for `content` and `iter_content`.
+  `requests.exceptions.SSLError` for `content` and `iter_content`. (#6057)
+- Fixed issue where invalid Windows registry entires caused proxy resolution
+  to raise an exception rather than ignoring the entry. (#6149)
+- Fixed issue where entire payload could be included in the error message for
+  JSONDecodeError. (#6036)
 
 **Deprecations**
 
-- ⚠️ Requests has officially dropped support for Python 2.7. ⚠️
-- Requests has officially dropped support for Python 3.6 (including pypy3).
+- ⚠️ Requests has officially dropped support for Python 2.7. ⚠️ (#6091)
```

Comment:
Thoughts on moving these to the top under a section "Breaking Changes"?

Reply:
Yeah, that sounds fine. We've historically had deprecations at the bottom but I'm fine moving them up for visibility.